source/store.go

```diff
@@ -270,7 +270,8 @@ func BuildWithConfig(ctx context.Context, source string, p ClientGenerator, cfg
 		if err != nil {
 			return nil, err
 		}
-		return NewCRDSource(crdClient, cfg.Namespace, cfg.CRDSourceKind, cfg.AnnotationFilter, cfg.LabelFilter, scheme)
+		startInformer := true
```

Comment:
Does it make sense to set this via flag? I think we should give users the option to turn it off via flag.

Reply:
Otherwise we could just add true into NewCRDSource function

Reply:
Isn't that what the --events flag is for? So instead of introducing a new flag, the startInformer parameter should be wired to the value of this flag.

Nevertheless, I would like to see this PR merged sooner than later, as we have to maintain our own build of external-dns as long as this is not part of the upstream binary 😄  So it would be nice if this would not block the merging of a PR that is open for over a year now.

Reply:
That makes even more sense, I totally agree 👍🏻

Reply:
Agreed, good idea.  I just pushed a commit to do that.  Thx @mgruener and @njuettner!

Reply:
(not sure what common practice is in this project, but I'll leave this conversation unresolved, and let @njuettner resolve it when he's satisfied).